tools/clang/include/clang/SPIRV/SpirvInstruction.h

```diff
@@ -2142,6 +2132,13 @@ class SpirvDebugValue : public SpirvDebugInstruction {
 
   bool invokeVisitor(Visitor *v) override;
 
+  SpirvDebugLocalVariable *getDebugLocalVariable() const { return debugVar; }
+  SpirvInstruction *getValue() const { return value; }
+  SpirvDebugExpression *getDebugExpression() const { return expression; }
+  llvm::SmallVector<SpirvInstruction *, 4> getIndices() const {
+    return indices;
+  };
+
```

Comment:
Again `SpirvDebugValue` class is not used ... if you want to remove it, I will do it.

Reply:
Removed